mmv1/products/securitycenterv2/FolderMuteConfig.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderMuteConfig'
+base_url: '{{parent}}/foldermuteConfigs'
+self_link: '{{name}}'
+create_url: '{{parent}}/foldermuteConfigs?foldermuteConfigId={{folder_mute_config_id}}'
+update_verb: :PATCH
+update_mask: true
+description: |
+  Mute Findings is a volume management feature in Security Command Center
+  that lets you manually or programmatically hide irrelevant findings,
+  and create filters to automatically silence existing and future
+  findings based on criteria you specify.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/folders.locations.muteConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_folder_mute_config'
```

Comment:
This needs to match the file name of the example, so `scc_v2_folder_mute_config_basic`

Reply:
addressed